src/main/resources/assets/gtce2oc/lang/en_us.lang

```diff
@@ -0,0 +1,3 @@
+itemGroup.gtce2oc=gtce2oc
```

Comment:
Tried this changes locally and noticed that localization files are not loading.

This file name apparently should be `en_US.lang`. It would be missed in case-sensitive filesystems otherwise.